setup_WOA_initial_conditions.py

```diff
@@ -93,8 +94,7 @@
     t_conservative = gsw.CT_from_t(s_absolute,t_in_situ,pressure_tile)
     
     # save to netcdf
-    #save_file = data_dir + 'woa13_decav_ts_jan_04v2.nc'
-    save_file = dst_data_dir + 'woa13_decav_ts_'+mon[mm]+'_04v2.nc'
+    save_file = dst_data_dir + 'woa23_decav_ts_'+mon[mm]+'_04.nc'
     print(save_file)
```

Comment:
It looks like these 12 'woa23_decav_ts_'+mon[mm]+'_04.nc' files get modified in place to add the time variable and the conservative_temperature, so we need to add metadata for this operation. e.g. what was done, how is practical salinity defined, what code was used, etc